integrations/clickup/clickup/client.py

```diff
@@ -0,0 +1,172 @@
+from loguru import logger
+from typing import Any, AsyncGenerator
+
+from port_ocean.utils import http_async_client
+
+from .utils import timestamp_to_datetime
+
+
+WEBHOOK_NAME = "Port-Ocean-Events-Webhook"
+WEBHOOK_EVENTS = [
+    "listCreated",
+    "listUpdated",
+    "listDeleted",
+    "taskCreated",
+    "taskUpdated",
+    "taskDeleted",
+    "taskCommentPosted",
+    "taskCommentUpdated",
+]
+
+
+class ClickupClient:
+    def __init__(self, clickup_personal_token: str) -> None:
+        self.clickup_url = "https://api.clickup.com/api/v2"
+        self.clickup_personal_token = clickup_personal_token
```

Comment:
You can use the clickup_personal_token directly in the header instead of adding it as a class variable